Project.toml

```
@@ -9,11 +9,22 @@ LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MKL_jll = "856f044c-d86e-5d09-b602-aeab76dc8ba7"
 Preferences = "21216c6a-2e73-6563-6e65-726566657250"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
+Requires = "ae029012-a4dd-5104-9daa-d747884805df"
```

I think the ChainRules dependency should be a regular dependency on Julia < 1.9. AbstractFFTs depends on it anyway, so it is already an indirect dependency and the rules would always be loaded via Requires anyway - but due to the deficiencies of Requires they could not be precompiled.
```suggestion
ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
```